src/generator-attributes.cs

```diff
@@ -712,7 +712,11 @@ public class NotImplementedAttribute : Attribute {
 // }
 [AttributeUsage (AttributeTargets.Interface, AllowMultiple=false)]
 public class CategoryAttribute : Attribute {
-	public CategoryAttribute () {}
+	public bool IgnoreStaticWarnings;
+	public CategoryAttribute (bool ignoreStaticWarnings = false)
```

Comment:
Not sure if it's an issue since it would affect the temporary file ? but its better (and easy) to fix it